Jenkinsfile

```diff
@@ -0,0 +1,366 @@
+                        git checkout ${DEFAULT_BRANCH}
+                        """
+        }
+        pullRequests.each {
+          repoName, pullRequest ->
+		  sh \
+          """
+                        cd zlux/${repoName}
+                        git fetch origin pull/${pullRequest['number']}/head:pr
+                        git merge pr
+                        """
+        }
+      }
+      stage("Set version") {
+        def (majorVersion, minorVersion, microVersion) = zoweVersion.tokenize(".")
+        sh \
+        """
+                    cd zlux/zlux-build
+                    sed -i -e "s/MAJOR_VERSION=0/MAJOR_VERSION=${majorVersion}/" \\
+                           -e "s/MINOR_VERSION=8/MINOR_VERSION=${minorVersion}/" \\
+                           -e "s/REVISION=4/REVISION=${microVersion}/" \\
+                            version.properties
+                    echo "Set version to:"
+                    cat version.properties
+                    cd ../zlux-app-server
+                    if [ -e "manifest.yaml" ]; then
+                      export commit_hash=\$(git rev-parse --verify HEAD)
+                      export current_timestamp=\$(date +%s%3N)
+					  export zlux_version="${majorVersion}.${minorVersion}.${microVersion}"
+                      sed -i -e "s|{{build\\.branch}}|${BRANCH_NAME}|g" \\
+                             -e "s|{{build\\.number}}|${BUILD_NUMBER}|g" \\
+                             -e "s|{{build\\.commitHash}}|\${commit_hash}|g" \\
+                             -e "s|{{build\\.timestamp}}|\${current_timestamp}|g" \\
+							 -e "s|{{build\\.version}}|\${zlux_version}|g" \\
+                             "manifest.yaml"
+                      echo "manifest is:"
+                      cat manifest.yaml
+                    fi
+
+        """
+      }
+      stage("Build") {
+        sh "bash -c 'cd zlux/zlux-build && set -o pipefail && ant testing 2>&1 | grep -vE \"^\\s+\\[exec\\]\\s+0\\% compiling\"'"
+      }
+      stage("Test") {
+        pullRequests.each {
+          repoName, pullRequest ->
+          if (repoName != "zlux-app-server") {
+            sh \
+            """
+                    cd dist
+                    packages=\$(find ./${repoName} -name package.json | { grep -v node_modules || true; })
+                    for package in \$packages
+                    do
+                        sh -c "cd `dirname \$package` && npm run test --if-present"
+                    done
+                    """
+          }
+        }
+        setGithubStatus(GITHUB_TOKEN, pullRequests, "success", "This commit looks good")
+		    sh "cd zlux/zlux-build && ant -Dcapstone=../../dist removeSource"
+	  }
+      stage("Package") {
+		sh \
+          """
+            chmod +x dist/zlux-build/*.sh
+            cd dist
+            tar cf ../zlux.tar -H ustar *
+            cd ..
+            git clone -b feature/tag-script https://github.com/1000TurquoisePogs/zowe-install-packaging.git
```

Comment:
you should change this to a tag of zowe, like tagged version 1.20
that way, if i delete mine, this will not break.
but at the same time, 1.20 wont disappear so it will be stable.